test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -104,13 +104,17 @@ class AndroidArgs(
     override val hasMultipleExecutions: Boolean get() = super.hasMultipleExecutions || additionalAppTestApks.isNotEmpty()
 
     init {
+        if (appApk == null) additionalAppTestApks.filter { (app, _) -> app == null }.let { missingApks ->
+            if (missingApks.isNotEmpty()) throw FlankFatalError("Cannot resolve app apk pair for ${missingApks.map { it.test }}")
```

Comment:
I am wondering if it wasn't be better to print just apk file name not full path to file. For many (potential) incorrect pairs it would be hard to read something.

Reply:
I agree a file name is easier to read. I think the full path will be useful when debugging problems in CI where the paths aren't obvious though. Maybe we could print each missing path on a new line to improve readability?